snuba/snuba_migrations/generic_metrics/0029_add_use_case_id_index.py

```diff
@@ -0,0 +1,76 @@
+from typing import Sequence
+
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.AddIndex(
+                storage_set=StorageSetKey.GENERIC_METRICS_COUNTERS,
+                table_name="generic_metric_counters_aggregated_local",
+                index_name="set_idx_use_case_id",
+                index_expression="use_case_id",
+                index_type="set",
```

Comment:
I opted for a `set` type here since use case IDs are very low cardinality, and our hardcoded in our repos, so they can't change easily.